src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,74 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public class Node<K , V>{
+        private int hash;
+        private K key;
+        private V value;
+        Node<K, V> next;
+
+
+        public Node(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+    }
+    private  Node<K, V> tail;
+
+    public static final int DEFAULT_CAPACITY = 16;
+
+    private Node<K, V>[] elements = new Node[DEFAULT_CAPACITY];
+
+    private int count;
 
     @Override
     public void put(K key, V value) {
+        Node<K, V> newNode = new Node<>(key,value);
+        if (count == 0){
+            elements[0] = newNode;
+            tail = newNode;
+        } else if (count > DEFAULT_CAPACITY){
```

Comment:
The condition `count > DEFAULT_CAPACITY` is incorrect for triggering a resize. Typically, a resize should occur when the number of elements reaches a certain load factor of the capacity, not when it exceeds the default capacity. Consider using a load factor to determine when to resize.